site/docs/python-quickstart.md

```diff
@@ -0,0 +1,40 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Examples
+
+## Inspect Table Metadata
```

Comment:
It would be good to have the information on how to install the library in a section here. In user-facing docs like this, we need to be clear that installing from master is for development and testing purposes. We can't recommend using code unless it is a released version. That means the wording should be something like "Iceberg for Python is not yet released and published to PyPI. To try out the python library, you can install it using `pip -e`: ..."

Reply:
added the following text:

> Iceberg python is currently in development, and as such, should __only__ be used for development and testing purposes until an official release has been made.
> 
> As such, we are not currently publishing to PyPi so the best way to install the library is to perform the following steps:

Let me know if that sounds ok

Reply:
Sounds a little scary to me. We just want to make it clear that this isn't how to use an official release.

Reply:
The new wording sounds good. Thanks!